corehq/warehouse/models/staging.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models, transaction
+
+from dimagi.utils.couch.database import iter_docs
+
+from corehq.apps.groups.dbaccessors import get_group_ids_by_last_modified
+from corehq.apps.groups.models import Group
+
+
+class StagingTable(models.Model):
+
+    staged_on = models.DateTimeField(auto_now_add=True)
```

Comment:
this is actually inherited by all tables that are staging tables. so for each group record in the staging table it will also include the date that it was added. just thought it might be useful meta data. will add a column for batch id, but probably won't address it in this pr until i sort out how the batch state will manifest